app/modules/sightings/resources.py

```diff
@@ -259,6 +272,18 @@ def post(self, args):
                 'Sighting.post imbalanced encounters in %r or %r'
                 % (request_in, result_data),
             )
+
```

Comment:
i had left this in because ben and i were (a while ago) using it as a test way to _directly_ get a post in (e.g. basically skipping to the real-object-commit part of our pipeline.  lemme check with ben and see how desirable this is (even for testing).  if we vote to kill it off, i will make it another ticket of its own.